TickSpec/TickSpec.fs

```diff
@@ -134,7 +134,6 @@ type StepDefinitions (givens,whens,thens,events,valueParsers) =
                         | step -> step
                     p,a,sm
                 )
-                |> Seq.distinctBy (fun (p,a,m) -> p)
```

Comment:
On the face of it, this seems a reasonable change.

It seems strange though - the sort of thing that could have been added to shut up some condition during development for some subtle case.

But, dropping random matching steps doesnt make any sense to me

Surprised more than one tag of each kind (G/W/T) to be permissible though - that should probably be restricted via AttributeUsage?

Not really sure what test cases I'd even put around it as valid ones...

Don't have an objection to the PR as it stands though personally

Reply:
> It seems strange though - the sort of thing that could have been added to shut up some condition during development for some subtle case.

Agree. It seems that it is there from beginning of time. I was trying to think what it will break, but it just does not make sense to me and it will potentially just hide issues. Like the one we have in tests. We should rather fail tests when there is ambiguity than to hide it and have "random" behaviour depending on the order how the attributes are returned by reflection.

> Surprised more than one tag of each kind (G/W/T) to be permissible though - that should probably be restricted via AttributeUsage?

I think that it potentially makes sense. You can support different wording of the step and map it to the same method (considering that the order of argument is the same).

> Don't have an objection to the PR as it stands though personally
👍

Reply:
>I think that it potentially makes sense. You can support different wording of the step and map it to the same method (considering that the order of argument is the same).

Don't know; while I can imagine it making for a less convoluted regex sometimes, it definitely smells like a misfeature (but I can't say I've looked at how other impls approach such scenarios).